src/commands/run-tests.yml

```diff
@@ -40,6 +40,11 @@ parameters:
       parameter requires a Parallel Computing Toolbox license.
     type: boolean
     default: false
+  split-type:
```

Comment:
Have we done a naming pass for `split-type`? How about `test-split-type` or `split-by` or `split-by-type`?

Reply:
I have raised a new PR where the parameter name has been changed.